data/stoa0089/stoa001/stoa0089.stoa001.perseus-lat2.xml

```diff
@@ -29,12 +29,14 @@
         <sourceDesc>
           <biblStruct>
             <monogr>
-              <title>Claudian, with an English translation, in Two Volumes, Vol. II</title>
-              <author>Claudius Claudianus</author>
-              <editor role="editor">Maurice Platnauer</editor>
+              <title>Claudian</title>
+              <title type="sub">with an English translation</title>
+              <author>Claudian</author>
```

Comment:
inconsistent with eng1 edition where `<author>Claudius Claudianus</author>` is used

Reply:
I am going to stick with Claudius Claudianus I think because that is what it is in the Worldcat record. Although it is never used in the book.